external/error_report/Kconfig

```diff
@@ -0,0 +1,50 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config ERROR_REPORT
+	bool "Error Report Module"
+	default n
+	---help---
+		enable Error Reporting
+
+if ERROR_REPORT
+
+config ERROR_REPORT_NENTRIES
+	int "Number of Error Records"
+	default 6
+	---help---
+		Maximum number of error records to store in memory
+		
+config ERROR_REPORT_DEBUG
+	bool "Debug Error Reporting"
+	default n
+	---help---
+		Debug Error Reporting
```

Comment:
You should make a dependancy with CONFIG_DEBUG

Reply:
Added dependency in Kconfig.